Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -209,8 +210,60 @@ CFArrayRef _DWriteCopyFontNamesForFamilyName(CFStringRef familyName) {
  * Note: This function currently uses a cache, meaning that fonts installed during runtime will not be reflected,
  * unless registered to the user font collection
  */
-CFStringRef _DWriteGetFamilyNameForFontName(CFStringRef fontName) {
-    return _DWriteGetFontPropertiesFromName(fontName)->familyName.get();
+CFStringRef _DWriteGetCompatibleFamilyName(CFStringRef fontName, IDWriteFontFace* fontFace) {
+    woc::unique_cf<CFStringRef> upperFontName(_CFStringCreateUppercaseCopy(fontName));
```

Comment:
GetFontPropertiesFromUppercaseFontName expects a unique_cf and changing that entire path is beyond the scope of this PR #ByDesign